include/hwinfo/cpu.h

```diff
@@ -11,6 +11,24 @@ namespace hwinfo {
 
 class Socket;
 
+#ifdef HWINFO_UNIX
+struct Jiffies
+{
+  Jiffies(){
+    working = -1;
+    all = -1;
+  }
+
+  Jiffies(const int64_t& _all, const int64_t& _working) {
```

Comment:
Just pass int64_t as Argument. The value of _all and _working is copied anyways when reassigning all and working. Passing a const int reference may be more expensive than just passing the value because of the dereferencing while copying an int is quite cheap.